pkg/pullrequest/scm.go

```diff
@@ -102,7 +102,8 @@ func githubHandlerFromURL(u *url.URL, token string, skipTLSVerify bool, logger *
 	// gosec complains that we're setting the InsecureSkipVerify option to bypass
 	// security checks. As long as this is generally set to false (which is the
 	// case by default), this should be fine.
-	t.TLSClientConfig = &tls.Config{InsecureSkipVerify: skipTLSVerify} // nolint: gosec
+	/* #nosec G402 */
```

Comment:
v minor: Use // #nosec G402 to be consistent with the rest of the commenting style.

Reply:
Sounds good. I have changed the commenting style.